.github/actions/spelling/candidate.patterns

```diff
@@ -0,0 +1,480 @@
+# marker to ignore all code on line
```

Comment:
This feature isn't actually available in 0.0.20, but will be when `main` updates to the next release.

You can see the results of this in the patterns file below...